tox.ini

```diff
@@ -1,20 +1,26 @@
 [tox]
-# not ideal, but nose doesnt work on > 3.9, so keep this
-# until we change the test infra
-envlist = py3{8,9}
+envlist = py3{8,9,10,11,12},format,lint
```

Comment:
If we drop python 3.8, remember to remove it from here as well. And in github actions too.